src/web/index.jsx

```diff
@@ -6,6 +6,8 @@ import Routes from 'aws/common/Routes';
 
 import AWSInputConfiguration from './aws/AWSInputConfiguration';
 import AWSCloudWatchApp from './aws/cloudwatch/CloudWatchApp';
+import SlackNotificationForm from './event-notifications/event-notification-types/SlackNotificationForm';
+import SlackNotificationSummary from './event-notifications/event-notification-types/SlackNotificationSummary';
```

Comment:
As with the Java code, I don't think there's any reason to so closely follow the package structure from the graylog2-server repository.  I would suggest moving these to a folder simply called `slack`

Reply:
@waab76 - The suggestion came from @bernd to use a similar structure in the server repository.